languages/ZunitConfig.groovy

```diff
@@ -75,14 +75,32 @@ buildUtils.createLanguageDatasets(langQualifier)
 // DSN=${props.zunit_bzureportPDS}(${member})
 """
 	if (props.codeZunitCoverage && props.codeZunitCoverage.toBoolean()) {
+	        // codeCoverageHost
+		codeCoverageHost = props.getFileProperty('zunit_CodeCoverageHost', buildFile)
+		if (props.codeCoverageHeadlessHost != null)
+			codeCoverageHost = props.codeCoverageHeadlessHost
```

Comment:
Coding best practices :-). Avoid calling code instruction for nothing. In some situation that can degrade performances.
This code (and following lines) should be:
```		
		if (props.codeCoverageHeadlessHost != null)
			codeCoverageHost = props.codeCoverageHeadlessHost
               else
			codeCoverageHost = props.getFileProperty('zunit_CodeCoverageHost', buildFile)
```